aquarius/events/events_monitor.py

```diff
@@ -172,7 +174,12 @@ def process_current_blocks(self):
             self.retry_mechanism.process_queue()
 
         last_block = self.get_last_processed_block()
```

Comment:
instead of the while True loop couldn't we do a isinstance(last_processed_block, int)? a while True nested in 2 try / except can have unexpected behaviors